assume/scenario/loader_amiris.py

```diff
@@ -408,7 +523,7 @@ async def load_amiris_async(
     # To use this with amiris run:
     # git clone https://gitlab.com/dlr-ve/esy/amiris/examples.git amiris-examples
     # next to the assume folder
-    scenario = "Simple"  # Germany2019 or Austria2019 or Simple
+    scenario = "Germany2019"  # Germany2019 or Austria2019 or Simple
     base_path = f"../amiris-examples/{scenario}/"
```

Comment:
shouldn't this be moved/copied to an example file to showcase how to use this feature?